dynamo-event-to-datacite-request/src/main/java/no/unit/nva/events/examples/DataciteDoiRequest.java

```diff
@@ -0,0 +1,157 @@
+package no.unit.nva.events.examples;
+
+import java.net.URI;
+import java.util.Objects;
+import no.unit.nva.events.handlers.JsonSerializable;
+import nva.commons.utils.JacocoGenerated;
+
+/**
+ * Example class. Will be deleted.
+ */
+public class DataciteDoiRequest implements JsonSerializable {
```

Comment:
Rename to ExampleClass , and keep it under `src/test/java/`

Reply:
I want to use it later for demo handlers

Reply:
@axthosarouris :   I just wanted to avoid confusing an example class with the real implementation when it arrives.

Reply:
If other modules should include it, probably want to publish it as a module with the name `test` or `example` in the artifact id.